frontend/workflows/projectSelector/src/hello-world.tsx

```diff
@@ -161,6 +157,10 @@ const ProjectSelector = () => {
 
     let allPresent = true;
     _.forEach(Object.keys(state[Group.PROJECTS]), p => {
+      /*
+      TODO: b/c of this conditional, if a user adds an upstream/downstream we already have the project data for
+      to the custom project group, allPresent will be true and we wont trigger an api call
+      */
```

Comment:
Whats the todo here?

Reply:
updated the todo to mention one approach we can take. one approach is updating the conditional to check if the project in state[project] is also in state[upstream]/state[downstream] but need to think on it a bit more